packages/clerk-js/src/ui/constants.ts

```diff
@@ -7,3 +7,8 @@ export const ORGANIZATION_PROFILE_NAVBAR_ROUTE_ID = {
   GENERAL: 'general',
   MEMBERS: 'members',
 };
+
+export const USER_BUTTON_ITEM_ID = {
+  MANAGE_ACCOUNT: 'manageAccount',
+  SIGN_OUT: 'signOut',
```

Comment:
I'm a bit skeptical about the camel case. I would prefer kebab case, tbh.

```suggestion
  MANAGE_ACCOUNT: 'manage-account',
  SIGN_OUT: 'sign-out',
```

Reply:
WDYT?

Reply:
Discussed offline with Pantelis and we decided that it's not worth